Craigslist/Craigslist.Business/ListingsManager.cs

```diff
@@ -17,17 +21,20 @@ public void PublishListing(Listing listing)
 			}
 		}
 
-		public List<Listing> GetListingsByCategoryId(long? categoryId)
+		public IPagedList<Craigslist.Domain.Entities.Listing> GetListingsByCategoryId(int? page, long? categoryId)
 		{
 			using (var domain = new CraigslistDomain())
 			{
+                int pageSize = Convert.ToInt32(ConfigurationManager.AppSettings["pageSize"]);
+                int pageNumber = (page ?? 1);
 				return domain
 					.Listings
 					.Include(l => l.Category)
 					.Include("Category.ParentCategory")
 					.Include(l => l.Contact)
-					.Where(listing => categoryId == null || listing.CategoryId == categoryId)
-					.ToList();
+                    .Where(listing => categoryId == null || listing.CategoryId == categoryId)
+                    .OrderBy(x => x.Created)
```

Comment:
for lambdas there're some naming conventions. Regularly they're called i, j, k or by first letter of the entity (l in my case means Listing) or by its full name (Listing).